packages/adblocker/src/engine/domains.ts

```diff
@@ -12,11 +12,19 @@ import { StaticDataView, sizeOfUint32Array, sizeOfUTF8 } from '../data-view.js';
 import { binLookup, hasUnicode, HASH_INTERNAL_MULT } from '../utils.js';
 
 export class Domains {
-  public static parse(parts: string[], debug: boolean = false): Domains | undefined {
+  public static parse(value: string, delimiter: string, debug = false): Domains | undefined {
```

Comment:
Maybe it would be best to introduce an object to allow extending the options in the future (`debug` could be the only value for now). Another benefit is that we force the user to put names on the arguments (`{ debug: true }` vs. `true`)

```typescript
public static parse(value: string, { delimiter, debug }: { delimiter?: string, debug?: boolean } = {}): Domains | undefined {
```

^ type to be adapted depending on the default values and which options are mandatory.

Reply:
fixed in [`95bb925` (#4525)](https://github.com/ghostery/adblocker/pull/4525/commits/95bb925d90c2b408c0db483c3632e93e8fc67d62)